kolibri/core/analytics/serializers.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework import serializers
+
+from .models import PingbackNotification
+from .models import PingbackNotificationDismissed
+
+
+class PingbackNotificationSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = PingbackNotification
+        fields = ('id', 'version_range', 'timestamp', 'link_url', 'i18n')
+
+
+class PingbackNotificationDismissedSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = PingbackNotificationDismissed
+        fields = ('user', 'notification')
```

Comment:
minor style note: I find separating the `serializers.py` and `api.py` into two files to be unhelpful indirection. They're tightly coupled so I'd recommend just moving these classes to `api.py`